docs/activities.md

```diff
@@ -39,6 +39,21 @@ Activity Definitions are named and referenced in code by their <a class="tdlp" h
 
 <div class="tdiw"><div class="tditw"><p class="tdit">Activity Definition</p></div><div class="tdiiw"><img class="img_ev3q" src="/diagrams/activity-definition.svg" alt="Activity Definition" height="680" width="1000" /></div></div>
 
+#### Idempotency
+
+Temporal recommends that Activities be idempotent.
+
+An Activity is idempotent if multiple <a class="tdlp" href="/tasks#activity-task-execution">Activity Task Executions<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is an Activity Task Execution?</span><br /><br /><span class="tdlppd">An Activity Task Execution occurs when a Worker uses the context provided from the Activity Task and executes the Activity Definition.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="/tasks#activity-task-execution">Learn more</a></span></span></a> do not change the state of the system beyond the first Activity Task Execution.
+
+An Activity can be tried more than once even if the Retry Policy says it shouldn't.
```

Comment:
per @tomwheeler 

> This will inevitably lead the reader to ask "Why? Under what conditions would an Activity be tried more than once, contradicting what I put in the RetryPolicy?" I think the documentation will be stronger if that question is briefly answered right there where it's brought up, then linking to relevant documentation that explains it in more detail.

I agree - 
I think it is in some rare occurrences during a failover? I would definitely follow up on this point.